tpm2/tpm2.go

```diff
@@ -1546,6 +1546,26 @@ func encodeCertify(parentAuth, ownerAuth string, object, signer tpmutil.Handle,
 	return concat(ha, auth, params)
 }
 
+// This function differs from encodeCertify in that it takes the scheme
+// to be used as an additional argument.
+func encodeCertifyEx(parentAuth, ownerAuth string, object, signer tpmutil.Handle, qualifyingData tpmutil.U16Bytes, scheme tpmtSigScheme) ([]byte, error) {
```

Comment:
As discussed in #210, can you rename these to `objectAuth` and `signerAuth` as you suggested, both here and in Certify?